tools/ramdump/ramdump_tool.c

```diff
@@ -228,7 +266,7 @@ static int ramdump_recv(int dev_fd)
 		printf("Receiving number of regions to be dumped failed, ret = %d\n", ret);
 		return -1;
```

Comment:
Because of several reasons, it fails to get dump like this line, line 281, line 299 and so on. But when it fails, it goes to `goto ramdump_err;` by line 482. It does not free allocated memory, mem_info. Could you check that? @thapav

Reply:
Updated as well.
Since `init_err:` now checks if `mem_info != NULL` before freeing, it is safe to `goto `the label in case `(ret != 0)`